nx-X11/programs/Xserver/Xext/xcmisc.c

```diff
@@ -40,11 +40,7 @@ from The Open Group.
 #include <nx-X11/extensions/xcmiscstr.h>
 #include "modinit.h"
 
-#if HAVE_STDINT_H
 #include <stdint.h>
-#elif !defined(UINT32_MAX)
-#define UINT32_MAX 0xffffffffU
-#endif
```

Comment:
There is a typo in the commit msg of 6b9d3db43626249672738dbd54ce5d7a9688f315: definde -> defined